.changeset/many-apricots-try.md

```diff
@@ -0,0 +1,6 @@
+---
+'@vocab/phrase': minor
+'@vocab/cli': minor
+---
+
+Vocab pull: throw error if a global key does not have translation in Phrase
```

Comment:
**suggestion:** It seems #166 had no changeset - would it be wise to add a note here about that as well? (eg, add support for global-key, or something to that effect)

Reply:
Good point Rico.

Reply:
good observation @rstacruz I was thinking of doing the changeset on the main feature branch so that we only have 1 changeset. I'm not too sure whether it was better doing multiple changesets or just the one from the main feature branch.